pkg/util/iptables/iptables.go

```diff
@@ -783,3 +789,90 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+// ParseError records the payload when iptables restore fails
+type ParseError interface {
+	// Line returns the number of lines on which iptables-restore is failing
+	// NOTE: First line is line 1
+	Line() int
+	// Error returns the error message of the location where the iptables-restore execution failed
+	Error() string
```

Comment:
"Error returns the error message of the parse error, including line number."

Reply:
done